latest-thread.html

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: Latest LOOT thread
+permalink: /latest-thread/
+redirect_to:
+  - http://forums.bethsoft.com/topic/1588424-rel-loot-thread-22/
+---
+
+<p>The latest thread about LOOT on the Bethesda forums can be found at:
```

Comment:
Sorry, I wasn't clear - I see

![screenshot 7](https://cloud.githubusercontent.com/assets/2430738/12808205/03573eac-cb0d-11e5-8144-bfc61d0ea61c.png)

So it looks like Jekyll is ignoring what you've written and doing its own thing instead.